### PR TITLE
Merge #92, #143 into develop

### PR DESCRIPTION
**Pull request by leite08**
Tickets: #92 #143

### Dependencies

- Upstream: none
- Downstream: none

### Description

No need to review the code changes, one might want to check the commits/merges, list of PRs, destination branch.

Merges these PRs into `develop`:
- https://github.com/metriport/metriport/pull/58
- https://github.com/metriport/metriport/pull/59
- https://github.com/metriport/metriport/pull/60
- https://github.com/metriport/metriport/pull/61
- https://github.com/metriport/metriport/pull/61
- https://github.com/metriport/metriport/pull/62
- https://github.com/metriport/metriport/pull/63
- https://github.com/metriport/metriport/pull/64
- https://github.com/metriport/metriport/pull/65

### Release Plan

- anytime

### CODE REVIEW

**Review by Goncharo — CHANGES_REQUESTED** [review]
Aside from the linting issues, looks fine

File: api/app/src/mappings/google/biometrics.ts [hunk 1]
```diff
@@ -62,23 +59,23 @@ export const mapToBiometrics = (googleBiometrics: GoogleBiometrics, date: string
...
-            min_pct: min_max.min_item
-          }
-        }
+            min_pct: min_max.min_item,
+          },
+        };
       }
 
-      if (data.dataSourceId = sourceIdHeartBpm) {
+      if (data.dataSourceId === sourceIdHeartBpm) {
```
Comment (leite08) [inline api/app/src/mappings/google/biometrics.ts:67]: @Orta21 just to make sure, here we want to compare, not assign, right?

File: api/app/src/mappings/google/index.ts [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
 export const getValues = (arr: GooglePoint, key: ValueKey = ValueKey.fpVal): number[] => {
   return arr.reduce((acc, curr) => {
-    const hasFpVal = curr.value.filter(val => val[key]);
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const hasFpVal = curr.value.filter(val => val[key]).map(val => val[key]!);
 
-    hasFpVal.forEach(val => {
-      acc.push(val[key]!)
-    })
+    hasFpVal.forEach(val => acc.push(val));
```
Comment (Goncharo) [inline api/app/src/mappings/google/index.ts:30]: This code is getting a bit funky now, can we just change this whole bit to something like:

``` typescript
    curr.value.forEach(val => {
      const fpVal = val[key];
      if (fpVal) acc.push(fpVal);
    });
```

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: api/app/src/mappings/google/index.ts (hunk 2 above)
Comment (Goncharo) [reply api/app/src/mappings/google/index.ts:30]: ^ @Orta21

File: api/app/src/mappings/google/biometrics.ts (hunk 1 above)
Comment (Goncharo) [reply api/app/src/mappings/google/biometrics.ts:67]: ^ good catch

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**